src/main/java/uk/gov/hmcts/probate/service/fee/FeeService.java

```diff
@@ -78,10 +80,14 @@ private URI buildUri(String event, String amount) {
             .queryParam("event", event)
             .queryParam("amount_or_volume", amount);
 
-        if (FEE_API_EVENT_TYPE_COPIES.equals(event)) {
+        if (FEE_API_EVENT_TYPE_COPIES.equals(event) && !featureToggleService.isNewFeeRegisterCodeEnabled() ) {
             builder.queryParam("keyword", feeServiceConfiguration.getKeyword());
         }
 
+        if (FEE_API_EVENT_TYPE_COPIES.equals(event) && featureToggleService.isNewFeeRegisterCodeEnabled()) {
+            builder.queryParam("keyword", "GrantWill");
+        }
```

Comment:
I think the cognitive complexity of these 2 ifs can be simplified? Eg.
```
if (FEE_API_EVENT_TYPE_COPIES.equals(event)) {
  if (featureToggleService.isNewFeeRegisterCodeEnabled()) {
    builder.queryParam("keyword", "GrantWill");
  } else {
    builder.queryParam("keyword", feeServiceConfiguration.getKeyword());
  }
}
```
I _feel_ like that's simpler to read but I dunno. What do you think?